ktxspec.adoc

```diff
@@ -1149,7 +1149,7 @@ the file is invalid.
 [NOTE]
 ====
 `dfdTotalSize` is included so that the KTX file contains a complete
-descriptor as defined in Chapter 4 "Khronos Data Format Descriptor"
+descriptor as defined in Chapter 4 "`Khronos Data Format Descriptor`"
```

Comment:
Why the backticks here? This is a chapter title. Why write it in fixed-width font?

Reply:
Backticks inside regular quotes convert the latter into curved quotes. Please compare the rendered output.